selfdiagnose.go

```diff
@@ -3,16 +3,28 @@ package main
 import (
 	"context"
 	"fmt"
+	"net/http"
 	"os"
 
 	"cloud.google.com/go/pubsub"
 	selfdiagnose "github.com/emicklei/go-selfdiagnose"
 	"github.com/emicklei/go-selfdiagnose/task"
 )
 
-func addSelfdiagnose() {
+func selfdiagnoseHandler() http.Handler {
```

Comment:
might be good idea to have a Handler() func in the selfdiagnose package.

Reply:
Vooruit :) https://github.com/emicklei/go-selfdiagnose/pull/7